test/integration/targets/ec2_vol_facts/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+unsupported
```

Comment:
It looks like testing this PR is blocked on CI updates that @s-hertel is working on.

We should wait for those updates rather than disabling the tests.